src/benchmarklib/benchmark_sql_executor.hpp

```diff
@@ -40,10 +40,11 @@ class BenchmarkSQLExecutor {
                        const std::shared_ptr<const Table>& actual_result_table,
                        const std::optional<const std::string>& description = std::nullopt);
   void _verify_with_sqlite(SQLPipeline& pipeline);
-  void _visualize(SQLPipeline& pipeline) const;
+  void _visualize(SQLPipeline& pipeline);
 
   const std::shared_ptr<SQLiteWrapper> _sqlite_wrapper;
   const std::optional<std::string> _visualize_prefix;
+  int _num_visualized_plans{0};
```

Comment:
Why `int` and not something unsigned?

Reply:
Didn't really put too much thought into it, will change.